### DISCUSSION TITLE
Aderyn Fatal Compiler bug !!

### DISCUSSION THREAD

**Discussion by VaibhavvDange** [Cyfrin/aderyn#750] [Q&A]
I’d like to raise a fatal compiler bug identified in Aderyn, which states: 
```
"error: Fatal compiler bug! 
This is a fatal bug in the Aderyn, sorry!"
```
 Could someone provide clarification on the cause of this issue and the expected timeline for its resolution? Thank you!

**Comment by TilakMaddy** [comment]
Sorry for late response. We require the full message printed in the console to help debug.
